gpytorch/means/fkl_means.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import gpytorch
+from torch.nn.functional import softplus
+from gpytorch.priors import NormalPrior
+
+class LogRBFMean(gpytorch.means.Mean):
+    """
+    Log of an RBF Kernel's spectral density
+    """
+    def __init__(self, hypers = None):
+        super(LogRBFMean, self).__init__()
+        self.register_parameter(name="constant", parameter=torch.nn.Parameter(0. * torch.ones(1)))
+        self.register_parameter(name="lengthscale", parameter=torch.nn.Parameter(-0.3*torch.ones(1)))
+
+        # register prior
+        self.register_prior(name='constant_prior', prior=NormalPrior(torch.zeros(1), 100.*torch.ones(1), transform=None),
+            param_or_closure='constant')
+        self.register_prior(name='lengthscale_prior', prior=NormalPrior(torch.zeros(1), 100.*torch.ones(1), transform=torch.nn.functional.softplus),
+            param_or_closure='lengthscale')
+
+    def forward(self, input):
+        # logrbf up to constants is: c - t^2 / 2l
+        out = self.constant - input.pow(2).squeeze(-1) / (2 * (softplus(self.lengthscale.view(-1)) + 1e-7) )
+        # print("output shape = ", out.shape)
+        return out
+
+class SM_Mean(gpytorch.means.Mean):
```

Comment:
`SpectralMixtureMean`. Also, the file name should match the class you are importing (e.g., `spectral_mixture_mean.py`.

Also, do these means make sense to use with GP models outside of with FKL? If so, living here is fine. If not, they might live directly with the kernel.